### PR TITLE
chore: add config to next/eslint to make it work in monorepo

### PR DESCRIPTION
**Pull request by pawk3k**

### CODE REVIEW

File: apps/nextjs-app/.eslintrc.js [hunk 1]
```diff
@@ -30,7 +35,7 @@ module.exports = {
   ],
   rules: {
     // https://github.com/vercel/next.js/discussions/16832
-    '@next/next/no-img-element': 'off',
+    // '@next/next/no-img-element': 'off',
```
[the remark this answers is not in the document]
Comment (pawk3k) [reply apps/nextjs-app/.eslintrc.js]: Sorry already fixed it @belgattitude , yes it's your repository i'm okey with it :). Next js eslint complain without settings that i provided https://nextjs.org/docs/basic-features/eslint#rootdir here is a documentation

[the remark this answers is not in the document]
Comment (belgattitude) [reply apps/nextjs-app/.eslintrc.js]: About next/image... in my dirty app examples I want to keep the ability to quickly use all kind of external sources :smile: 

On the other hand, I was disappointed by next/image in the past (I don't say it's bad). 

I'll follow the next/future/image from nextjs 12.2 and see how it goes. https://nextjs.org/blog/next-12-2#improvements-to-nextimage and probably will enforce in the future.

Thanks for the rootDir, it's an amazing fix.

**Comment by belgattitude** [comment]
Thank you so much @pawk3k. Can I ask you 2 little things

1. Add a changeset with a patch for package '@your-org/eslint-config-bases' ?

It's pretty simple: `yarn g:changeset` and answer few questions. That will help will release and automated changelogs.

2. Add a note in eslint-config-bases README 

Can be a comment in https://github.com/belgattitude/nextjs-monorepo-example/tree/main/packages/eslint-config-bases#usage.

Maybe something like

```typescript
  /** If using plugin:'@next/next/core-web-vitals' and set the app folder
  settings: {
    next: {
      rootDir: 'apps/nextjs-app',
    },
  },
  */
```

File: apps/nextjs-app/.eslintrc.js [hunk 2]
```diff
@@ -14,6 +14,11 @@ module.exports = {
     project: 'tsconfig.json',
   },
   ignorePatterns: [...getDefaultIgnorePatterns(), '.next', '.out'],
+  settings: {
+    next: {
+      rootDir: 'apps/nextjs-app',
+    },
```
Comment (belgattitude) [inline apps/nextjs-app/.eslintrc.js:20]: Out of curiosity would it work with

```
  settings: {
    next: {
      rootDir: __dirname,
    },
  },
```

? 

That would make it easy to copy/paste

**Comment by belgattitude** [comment]
@pawk3k finally I had time to test it out.  Actually the nextroot is not needed when the eslint config is present in the app directory. 

It's only required if the eslint config is in the monorepo root folder.

That said I don't really know what would be best.  Either to close,  either to add a comment. 

Wdyt ?

**Comment by belgattitude** [comment]
I'll close but feel free to reopen

**Pull request closed without merging**